nrf_security/Kconfig

```diff
@@ -97,60 +121,35 @@ config MBEDTLS_CIPHER_AES_256_CTR_C
 config MBEDTLS_CIPHER_AES_256_CCM_C
 	bool
 
-comment "Configuration section"
+config MBEDTLS_CIPHER_AES_256_CFB_C
+	bool
 
-menuconfig AES_CONFIGURATION
+config MBEDTLS_CIPHER_AES_256_OFB_C
 	bool
-	prompt "AES   - Advanced Encryption Standard"
-	depends on CC310_BACKEND || MBEDTLS_VANILLA_BACKEND
-	default y
-	help
-	  This setting will enable support for AES and allow fine grained
-	  configuration of specific cipher support
 
-if AES_CONFIGURATION
+comment "Configuration section"
 
-if !NRF_CRYPTO_BACKEND_COMBINATION_0
+menu "AES   - Advanced Encryption Standard"
```

Comment:
I think changing this from menuconfig to menu is a step back in usability, as users must now enter each submenu to see if the group is configured or not.
The menuconfig allows user to completely disable AES if it is not needed. (AES is not best example, but comment also applies to ECC, RSA, AEAD etc. and thus all menu should share same look-n-feel)

Reply:
Fixed after some discussion